accession-commons-jpa/src/test/resources/features/retrieval.feature

```diff
@@ -22,8 +22,7 @@ Feature: Accession retrieval
 
   Scenario: I want to get a specific accession that doesn't exist in the system.
     When user retrieves accessions: id-service-A
-    Then operation finished correctly
-    And user receives no data
+    Then user should receive 'accession does not exist'
```

Comment:
IIRC these were auto-generated into code, but I can't see them reflected anywhere in this PR. Could you please check?

Reply:
The autogenerated runners will be in target folder, other than that the step definitions are all manual only .